test/cdk_diff.bats

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bats
 
-load "cdk_diff.sh"
+load "../cdk_diff.sh"
```

Comment:
Hrm, why is this needed now? Something change in `bats`?

Reply:
honestly i'm not sure 
i tried to look at last action run for previous version but it's gone (more than a year) 

i did update bats library, there can be internal change in that, or ubuntu-latest image can behave a bit differently after a year ... i guess were never going to be sure